testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt

```diff
@@ -461,8 +464,13 @@ class MiniAppDisplayActivity : BaseActivity(), PreloadMiniAppWindow.PreloadMiniA
                     NativeEventType.EXTERNAL_WEBVIEW_CLOSE,
                     "External webview closed"
                 )
-                if (!isClosedByBackPressed)
+                if (!isClosedByBackPressed) {
                     sampleWebViewExternalResultHandler.emitResult(intent)
+                }
```

Comment:
Usually, "isClosedByBackPressed" is triggered from `WebViewActivity` with `true` which means we can close the respected screens if there is nothing to emit result. So, I think, we can do following:

```suggestion
                } else finish()
```

Then, we don't need any other changes e.g. `isFromMiniAppByUrlActivityTag`

Reply:
Nice one then, thanks for confirming. I can go back to use this one once get at least one approval